### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by jssol**
### Project requirements

- Deleted all the boilerplate from CRA (text, images, styles).
- Created a directory called components.
  - Created a new Calculator.js file.
    - Create a React component that matches the design

- Didn't add any extra design.
- Developed just the UI of the calculator.
- Used class based components.

### Useful links
- [Full requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import './scss/App.scss';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```
Comment (BrianSammit) [inline src/App.js]: - [x] It would be better if your `App` component is class-based as one of the requirements. Kindly fix this minor issue. 😄 👍

**Review by BrianSammit — CHANGES_REQUESTED** [review]
# CHANGES REQUIRED 🔴 🛑
Hello @jssol, great job you are doing so far in this project. 💯

TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 

However, there are a few changes to make to move to the next project 🛠️ . Keep working hard 💪 💻 

## Required Changes ♻️
See my inline comment to know what changes are needed. 👇 👀 

Please make these required changes and send it again for another review 👍 💪
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
# HAPPY CODING!!! 😄 

cat


**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

--------

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/App.js (hunk 1 above)
Comment (jssol) [reply src/App.js]: For this one, I was getting a linter error as it is a stateless component.
@BrianSammit

Comment (jssol) [reply src/App.js]: Here is the rule it is bringing up `react/prefer-stateless-function`

**Review by rustam-tolipov — APPROVED** [review]
## Hello @jssol 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**